app/controllers/arrival/UnloadingPermissionPDFController.scala

```diff
@@ -49,14 +50,17 @@ class UnloadingPermissionPDFController @Inject()(
             arrivalMovementConnector.getPDF(arrivalId, token.value).flatMap {
               result =>
                 result.status match {
-                  case OK =>
+                  case OK => {
+
+                    val headers: Seq[(String, String)] = result.headers map {
+                      h =>
+                        (h._1, h._2.head)
+                    } toSeq
```

Comment:
Wouldn't this return all headers to the upstream? Should we only be passing the 2 that we know we want to pass on?

A failing test for that would probably be: request headers should contain CONTENT_TYPE and CONTENT_DISPOSITION but not random header ABCDE